nixos/modules/services/misc/sourcehut/default.nix

```
@@ -77,6 +77,14 @@ let
       type = types.path;
       apply = s: "<" + toString s;
     };
+    api-origin = mkOption {
+      description = lib.mdDoc "Origin URL for API, 100 more than web.";
```

I know you just moved this, but could we change this description? It doesn't inherently make sense. Anyone looking at this description will also be able to see the default, so I propose we just remove the last part.

```suggestion
      description = lib.mdDoc "Origin URL for the API";
```